cypress/integration/2.1-rest.apis.spec.js

```diff
@@ -4,7 +4,7 @@ import {
   updateProductStatus,
 } from '../support/availability-notify.apis'
 import { testSetup } from '../support/common/support'
```

Comment:
Drop the usage of testSetup and use loginViaCookies() which is available in support.js

Reply:
Please do this everywhere